src/components/ContentMapping/ContentMapping.ts

```diff
@@ -5,8 +5,8 @@ import { Stub, StubEditor } from '../ContentWidgets/_Stub/Stub';
 import { ContentSingularData } from '../_data/ContentSingularData';
 import { ExampleImage } from '../ContentWidgets/ExampleImage/ExampleImage';
 import { ExampleImageEditor } from '../ContentWidgets/ExampleImage/ExampleImageEditor';
-import { TodoList } from '../ContentWidgets/ToDoList/TodoList';
-import { TodoListEditor } from '../ContentWidgets/ToDoList/TodoListEditor';
+import { TodoList } from '../ContentWidgets/TodoList/TodoList';
```

Comment:
Careful with directory renaming. Git actually doesn't detect it easily. The best way to change the directory casing is to rename the directory to something else entirely, push that, then change it to what you wanted, then push again. Right now if I were to pull it I think it would crash.